javaharness/java/arcs/android/AndroidArcsEnvironment.java

```diff
@@ -41,7 +38,9 @@
   private static final Logger logger =
     Logger.getLogger(AndroidArcsEnvironment.class.getName());
 
-  private static final String ASSETS_PREFIX = "https://$assets/";
+  private static final String ASSETS_MACRO = "https://$assets/";
+  private static final String APK_ASSETS_URL_PREFIX = "file:////android_asset/arcs/";
+  private static final String ROOT_MANIFEST_FILENAME = "Root.arcs";
```

Comment:
Super minor nit: `root.arcs` seems better to me :P

Reply:
Manifest names were supposed to be Upper camel case:
https://docs.google.com/document/d/1QgvAX8M8s3g16zbICaK2NPqoTdma9dSxgJi6HnZRw3I/edit